CHANGELOG.md

```diff
@@ -6,70 +6,52 @@ The `next` version of 4.19 is now available.  Planned release date is April 2021
+- Enhanced the Gamepad so that navigation will only work over https.
+
+## Deprecated classes, properties, methods, and events
 
 The following are deprecated and will be removed in a future release:
 
-* BookmarksViewModel.BookmarkCreationOptions.captureExtent deprecated since version 4.17. Use captureViewpoint instead.
-* decorators.cast(classFunction) deprecated since version 4.14. Parameter decorators won't be supported by JavaScript decorators.
-* Extending multiple classes deprecated since version 4.13. Create mixins with TypeScript and JavaScript instead.
-* LabelClass.labelExpressionInfo.value deprecated since version 4.5. Use expression instead.
-* SceneView.constraints.collision deprecated since version 4.8. Use Ground.navigationConstraint instead.
-* SmartMapping.params.basemap deprecated since version 4.13. Use view instead.
-
-and
+| Class/Property/Method/Event | Alternate option | Version deprecated |
+|----------|-------------|--------------------|
+| `ProjectParameters.outSR` | [ProjectParameters.outSpatialReference](../api-reference/esri-tasks-support-ProjectParameters.html#outSpatialReference) | 4.4 |
+| `BasemapToggle.titleVisible` | [BasemapToggle.visibleElements.title](../api-reference/esri-widgets-BasemapToggle.html#visibleElements) | 4.15 |
+| `AreaMeasurement2DViewModel.clearMeasurement()` | [AreaMeasurement2DViewModel.clear()](/api-reference/esri-widgets-AreaMeasurement2D-AreaMeasurement2DViewModel.html#clear) | 4.16 |
+| `AreaMeasurement2DViewModel.newMeasurement()` | [AreaMeasurement2DViewModel.start()](/api-reference/esri-widgets-AreaMeasurement2D-AreaMeasurement2DViewModel.html#start) | 4.16 |
+| `AreaMeasurement3DViewModel.clearMeasurement()` | [AreaMeasurement3DViewModel.clear()](/api-reference/esri-widgets-AreaMeasurement3D-AreaMeasurement3DViewModel.html#clear) | 4.16 |
+| `AreaMeasurement3DViewModel.newMeasurement()` | [AreaMeasurement3DViewModel.start()](/api-reference/esri-widgets-AreaMeasurement3D-AreaMeasurement3DViewModel.html#start) | 4.16 |
+| `DirectLineMeasurement2DViewModel.clearMeasurement()` | [DirectLineMeasurement2DViewModel.clear()](/api-reference/esri-widgets-DirectLineMeasurement2D-DirectLineMeasurement2DViewModel.html#clear) | 4.16 |
+| `DirectLineMeasurement2DViewModel.newMeasurement()` | [DirectLineMeasurement2DViewModel.start()](/api-reference/esri-widgets-DirectLineMeasurement2D-DirectLineMeasurement2DViewModel.html#start) | 4.16 |
+| `DirectLineMeasurement3DViewModel.clearMeasurement()` | [DirectLineMeasurement3DViewModel.clear()](/api-reference/esri-widgets-DirectLineMeasurement3D-DirectLineMeasurement3DViewModel.html#clear) | 4.16 |
+| `DirectLineMeasurement3DViewModel.newMeasurement()` | [DirectLineMeasurement3DViewModel.start()](/api-reference/esri-widgets-DirectLineMeasurement3D-DirectLineMeasurement3DViewModel.html#start) | 4.16 |
+| `PrintTemplate.preserveScale` | [PrintTemplate.scalePreserved](../api-reference/esri-tasks-support-PrintTemplate.html#scalePreserved) | 4.16 |
```

Comment:
This table should go under the breaking changes section - these have already been deprecated at a previous release and have now been removed from the API. The old list of deprecations is still valid.

Reply:
I'll also remove the hyperlinks.